db/migrations/versions/ba5e693514cd_.py

```diff
@@ -1,42 +0,0 @@
-"""empty message
```

Comment:
Looks like the previous migrations have been accidentally deleted. Please can we restore them?